docs/src/modules/ROOT/pages/using-timefold-solver/modeling-planning-problems.adoc

```diff
@@ -1367,6 +1367,91 @@ public class Customer {
     public void setNextCustomer(Customer nextCustomer) {...}
 ----
 
+[#cascadeUpdateVariable]
+==== Cascade element update shadow variable
```

Comment:
This documentation is correct, but I've been wondering...
Should we rather (or maybe in addition) write a short section on "updating tail chains"? 
And maybe we don't hide it deep in a subsection on variable listeners.

Because as a user, that's what I will look for - not "cascade element update shadow variable".

Reply:
I've moved the contento into a separate section and renamed the title. I believe it should be placed after `List variable shadow variables` because it relies on some shadow variables discussed in that section.